dev-util/statifier/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please merge this into the original commit. `metadata.xml` is obligatory for packages, so that commit alone is not valid.

Reply:
All right, will be fixed with my next commit.